cmd/nerdctl/run.go

```diff
@@ -390,9 +389,9 @@ func createContainer(cmd *cobra.Command, ctx context.Context, client *containerd
 	}
 
 	var (
-		opts  []oci.SpecOpts
-		cOpts []containerd.NewContainerOpts
-		id    = idgen.GenerateID()
+		options []oci.SpecOpts
```

Comment:
I think this change is not needed.
If the linter complains we can add `// nolint: stylecheck` to ignore.